static/src/inline-svgs/icon/share-whatsapp.svg

```diff
@@ -0,0 +1,13 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
you need to remove these `xmlns` attributes to [stop IE8 going insane](http://stuntbox.com/blog/2013/06/bulletproof-inline-svg/)